src/js/Audit.js

```diff
@@ -78,6 +83,12 @@ axs.AuditConfiguration = function() {
      */
     this.showUnsupportedRulesWarning = true;
 
+    for (var prop in this) {
+        if ((this.hasOwnProperty(prop)) && (prop in config)) {
+            this[prop] = config[prop];
```

Comment:
This is cool, but is not going to work for `severity`, `ignoreSelectors` or `ruleConfig`. We'll probably need to special case those.